packages/compass-components/src/components/file-input.tsx

```diff
@@ -0,0 +1,180 @@
+/** @jsx jsx */
+import React from 'react';
+import path from 'path';
+import { css, jsx } from '@emotion/react';
+
+import { Button, Icon } from '..';
+
+const formItemHorizontalStyles = css`
+  margin: 15px auto 15px;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const formItemVerticalStyles = css`
+  margin: 5px auto 20px;
+`;
+
+const formItemErrorStyles = css`
+  border: 1px solid red;
+
+  &:focus {
+    border: 1px solid red;
+  }
+`;
+
+const buttonStyles = css`
+  width: 100%;
+`;
+
+const buttonErrorStyles = css`
+  border: 1px solid red;
+`;
+
+const labelHorizontalStyles = css`
+  width: 70%;
+  text-align: right;
+  padding-right: 15px;
+  margin: auto;
+`;
+
+const labelErrorStyles = css`
+  color: red;
+`;
+
+const labelIconStyles = css`
+  display: inline-block;
+  font: normal normal normal 14px/1 FontAwesome;
+  font-size: inherit;
+  text-rendering: auto;
+  margin: 0 0 0 5px;
+  cursor: pointer;
+  color: #bfbfbe;
```

Comment:
I left it for now because that's the little `(i)` icon that's used all over the rest of the connect form and maybe other places and I thought it would be a bit inconsistent until we move everything else over to leafygreen soon.